cmake/platform/freebsd/rbpi.cmake

```diff
@@ -0,0 +1,2 @@
+include(cmake/platform/linux/rbpi.cmake)
+list(APPEND ARCH_DEFINES -D_ARMEL -DTARGET_RASPBERRY_PI)
```

Comment:
`-D_ARMEL -DTARGET_RASPBERRY_PI` is now in cmake/platform/linux/rbpi.cmake so this can be removed